src/pybind/mgr/cephadm/inventory.py

```diff
@@ -175,6 +175,7 @@ def __init__(self, mgr):
         self.daemon_config_deps = {}   # type: Dict[str, Dict[str, Dict[str,Any]]]
         self.last_host_check = {}      # type: Dict[str, datetime.datetime]
         self.loading_osdspec_preview = set()  # type: Set[str]
+        self.etc_ceph_ceph_conf_refresh_queue: List[str] = []
```

Comment:
why do you change the kind of annotation?

Reply:
cause I didn't want to change the other unrelated lines.

Reply:
So ... should we avoid to use this kind of annotation?
`self.etc_ceph_ceph_conf_refresh_queue = []  # type: List[str]`

Reply:
this syntax:
```
var: int = 0
```
was added recently in Python 3.6. So, yes. for the mgr module, I'd avoid the comment-style annotations.